### ISSUE TITLE
Error - sb-callback.php

### ISSUE THREAD

**Issue by rzrhome** [sbpp/sourcebans-pp#266]
```
Error: the XML response that was returned from the server is invalid. Received: Warning: Illegal string offset 'type' in /var/customers/webs/sourcebans/includes/sb-callback.php on line 2142 Warning: Illegal string offset 'type' in /var/customers/webs/sourcebans/includes/sb-callback.php on line 2142 You have whitespace in your response.
```

Error comes by edit the group in "Group management"

**Comment by D4rk56** [comment]
Same error

**Comment by Kammi61871** [comment]
Same Error

**Comment by Groruk** [comment]
Fixed with 2a38a4f

**Comment by galexrt** [comment]
Same error pls fix
